### DISCUSSION TITLE
Unable to feedback on a kata solution?

### DISCUSSION THREAD

**Discussion by roothebear** [codewars/codewars.com#2548] [Q&A]
Hello!

I wonder if someone can assist, I'm trying to leave feedback on a kata solution however clicking the 'post' button doesn't appear to do anything. I've checked the documentation to see if it is a permission issue based on my honour level/rank but cannot see anything obvious, similarly searching the forums here.

If I have missed something obvious and you know what it is please do say, thank you!

Roo

`screenshot removed because containing spoiler information`

**Comment by hobovsky** [comment]
You missed #2493 ;)

**Reply by roothebear** [reply]
Thanks! It wasn't the best joke anyway.... :)
